src/Feersum/ParseRepl.fs

```diff
@@ -0,0 +1,30 @@
+module Feersum.ParseRepl
+
+open System
+open Feersum.CompilerServices.Diagnostics
+open Feersum.CompilerServices.Syntax.Tree
+open Feersum.CompilerServices.Syntax
+open Feersum.CompilerServices.Syntax.Parse
+
+let private read () =
+    ReadLine.Read("[]> ")
+    |> Parse.readProgram "repl.scm"
+
+let private print (result: ParseResult<Program>) =
```

Comment:
Some kind of ast explorer which allows you to cd and ls around the tree might be nice.